packages/gravity-ui-nunjucks/__tests__/a11y.spec.js

```diff
@@ -0,0 +1,73 @@
+const AxeBuilder = require('axe-webdriverjs');
+const selenium = require('selenium-webdriver');
+const chrome = require('selenium-webdriver/chrome');
+const { getComponentsNames, getViolations } = require('./_helpers');
+const pkgEnvs = require('../build/envs');
+const pkgPaths = require('../build/paths');
+
+const excludedFiles = [
+  '_preview',
+  'about',
+  '404-error-page',
+  'navigation',
+  'article',
+  'symbols',
+  'svg-symbols',
+  'image',
+  'inline-svg-symbol',
+  'logotype',
+  'logo',
+  'wiprodigital',
+  'designit',
+  'icon',
+];
+
+describe('A11y tests', () => {
+  let driver;
+  const components = getComponentsNames(pkgPaths.srcComponentsPath(), excludedFiles, '.njk');
+
+  components.forEach((component) => {
+    fdescribe(`${component} component`, () => {
```

Comment:
I think we need to replace fdescribe to describe as it will lock the tests to this describe block only.